src/vma/sock/sockinfo_udp.cpp

```diff
@@ -672,18 +672,19 @@ int sockinfo_udp::on_sockname_change(struct sockaddr *__name, socklen_t __namele
 	if (is_bound_modified && bound_port != INPORT_ANY) {
 
 		// Attach UDP unicast port to offloaded interface
-		// 1. Verify not binding to MC address in the UC case
```

Comment:
We want to create HW flow steering rules only once we have a L4 port defined. the IP address will be the unicats interface address or the MC address.
So the code only verifies the bound PORT != 0, so we can proceed with HW steering definitions.